modules/getting-started/pages/package-and-deploy-sca-application.adoc

```diff
@@ -0,0 +1,49 @@
+= Package and deploy your Application
+:description: Learn how to package and deploy your project.
+
+In this following sections, you will learn how to package and deploy your application as a local docker container.
+
+Be careful to following the xref:build:application-build.adoc#prerequisites[prerequisites] before starting this tutorial.
+
+== Application build
+
+=== Default application packaging
+
+In application mode packaging, the xref:runtime:user-application-overview.adoc[user application] and xref:runtime:admin-application-overview.adoc[admin application] are not embedded by default in the final project artifact.
+
+You need add them manually if you want to deploy them. To do it:
+
+* From Studio, Open Marketplace and filter by application type
+* Add the desired default applications to your project
+* In the pom.xml file, you will see the new dependencies added for the application
+
+=== Packaging as a docker image
+
+To package your application as a docker image, you need to run this commands at the root of your project:
+
+[source,shell]
+----
+  mvn clean package -Pdocker -Dbonita.environment=local
+----
+
+The docker image is created, you can find it with `docker images` command (the image name is: `getting-started-tutorial-local`).
+
+For more information about packaging as application mode, you can check xref:build:application-build.adoc[Application build section].
+
+== Deploy your application
+
+Run you docker application with command `docker run --rm --name GettingStarted -h GettingStarted -e -d -p 9000:8080 getting-started-tutorial-local:3.0.0-SNAPSHOT`
```

Comment:
suggestion:
Might not run with `--rm` to help debug failed container.
 `-h <hostname>` is required if you run an SP container but with the previous `mvn package` it implicitly packages a Community version. Is this what we want?
`-e` is not necessary 
```suggestion
Run you docker application with command `docker run --name GettingStarted -d -p 9000:8080 getting-started-tutorial-local:3.0.0-SNAPSHOT`
```